src/functTest/groovy/com/bmuschko/gradle/docker/DockerWorkflowFunctionalTest.groovy

```diff
@@ -457,6 +457,45 @@ class DockerWorkflowFunctionalTest extends AbstractGroovyDslFunctionalTest {
         result.output.contains("RestartPolicy : on-failure:999")
     }
 
+    def "Can build an image, create a container and set pid"() {
```

Comment:
Can we please move this test into `DockerCreateContainerFunctionalTest.groovy`? You will find that there's plenty of setup code already you can reused. I want to remove this class in the long run and distribute non-duplicate test cases to the relevant test classes.

Reply:
I moved the test into DockerCreateContainerFunctionalTest.groovy